kuma/wiki/tests/test_models.py

```diff
@@ -601,19 +601,24 @@ def test_correct_based_on_to_current_revision(self):
     def test_previous(self):
         """Revision.previous should return this revision's document's
         most recent approved revision."""
-        rev = revision(is_approved=True, save=True)
-        eq_(None, rev.previous)
-        # wait a second so next revision is a different datetime
-        time.sleep(1)
+        rev = revision(is_approved=True, created=datetime(2017, 4, 15, 9, 23),
```

Comment:
I'm trying to change as little as possible, and I consider a big re-write of``get_previous`` out of scope.  It is true that IDs increase for recent revisions, but that would need to be validated for older revisions in the database.